crates/core/tedge_agent/src/agent.rs

```diff
@@ -163,7 +175,7 @@ impl Agent {
         let signal_actor_builder = SignalActor::builder(&runtime.get_handle());
 
         // Health actor
-        let health_actor = HealthMonitorBuilder::new(TEDGE_AGENT, &mut mqtt_actor_builder);
+        let health_actor = HealthMonitorBuilder::new("health-actor", &mut mqtt_actor_builder);
```

Comment:
No, the name which is expected here is the name of the service.

This is one of the reasons why the system tests are failing. They expect the `"tedge-agent"` to be up, not some unknown `"health-actor"`. 

However, using `"tedge-agent"` here can only be a short term solution. To have the agent running concurrently on  several devices, one needs instance specific health status. The better would be to provide here the service topic id. Ignoring the issue with custom device topic id, the service topic id can be forged after the parent topic id.  For instance `device/child001/service/tedge-agent`.